src/guides/v2.3/install-gde/prereq/merch_install.md

```diff
@@ -21,9 +21,9 @@ If you don't use GoDaddy, you might need to contact your hosting provider's tech
 
 Magento requires the following software to run:
 
-*  Web server: Apache 2.2 or 2.4
-*  Programming language: [PHP](https://glossary.magento.com/php) 5.6.x or 5.5.x
-*  Database: MySQL 5.6.x
+*  Web server: Apache 2.4
+*  Programming language: [PHP](https://glossary.magento.com/php) 7.3.x or 7.4.x
```

Comment:
Will Magento 2.3.x be compatible with PHP 7.4 when Magento 2.4.0 releases?

Reply:
Good point.